tutorial/installing-appliance.md

```diff
@@ -18,7 +18,7 @@ This tutorial guides you through the steps that are required to install and init
 Make sure you have the following prerequisites:
 
 * An Ubuntu SSO account. If you don't have one yet, [create it](https://login.ubuntu.com).
-* A virtual or bare metal machine running Ubuntu 20.04 or 22.04. See the detailed {ref}`ref-requirements`.
```

Comment:
See above

Reply:
As per Simon's reply above, we will keep the removal of focal support in explicit mentions.

Reply:
Yes, that LGTM.
Thanks